src/content/translations/fr/contributing/design-principles/index.md

```diff
@@ -0,0 +1,94 @@
+- **Un tremplin :** nous sommes un premier pas, un tremplin pour tous ceux qui recherchent des réponses. Nous ne voulons pas remplacer ou devenir un substitut des nombreuses ressources qui existent déjà. Nous donnons des réponses et indiquons des pistes sérieuses pour la suite.
+
+Voyages de l'utilisateur • Contenu
+
+### 4. Crédible {#4-credible}
+
+Les gens peuvent chercher à découvrir l'écosystème Ethereum ou ils peuvent être sceptiques. Reconnaissez cette responsabilité dans la façon dont vous communiquez. Assurez-vous que ces deux catégories de personnes quitteront la page avec une plus grande confiance dans l'écosystème Ethereum.
+
+- **Actuel :** toujours à jour.
+- **Apportant une preuve sociale :** montre la taille, la diversité et l'activité de l'écosystème.
+- **Cohérent :** la cohérence dans la conception et le contenu assure la crédibilité.
+
+Direction visuelle • Contenu
+
+### 5. Amélioration collaborative {#5-collaborative-improvement}
+
+Le site web est le produit de nombreux contributeurs, tout comme l'écosystème dans son ensemble.
+
+- **Ouvert :** célébrez la transparence du code source, des processus et des projets dans l'ensemble de l'écosystème.
+- **Extensible :** la modularité est un élément clé derrière tout ce que nous faisons et les contributions devraient donc également être modulaires. La conception de base, le code des composants et la mise en œuvre du site devraient permettre de l'étendre facilement à l'avenir.
+- **Expérimental :** nous expérimentons, testons et itérons en permanence.
+- **Collaboratif :** ce projet nous réunit tous.
+- **Durable :** préparer une maintenance à long terme par la communauté.
+
+Vous pouvez voir nos principes de conception en action [dans l'ensemble de notre site](/).
+
+## Donnez-nous votre avis {#give-feedback}
+
+**Partagez vos commentaires sur ce document !** L'un des principes que nous proposons est l'« **amélioration collaborative** », ce qui signifie que nous voulons que le site web soit le produit de nombreux contributeurs. Dans l'esprit de ce principe, nous voulons donc partager ces principes de conception avec la communauté Ethereum.
+
+Bien que ces principes soient axés sur le site web ethereum.org, nous espérons que beaucoup d'entre eux sont représentatifs des valeurs de l'écosystème Ethereum dans son ensemble (par exemple, vous pouvez voir l'influence des [principes du livre blanc Ethereum](https://github.com/ethereum/wiki/wiki/White-Paper#philosophy)). Peut-être voudrez-vous même incorporer certains d'entre eux dans votre propre projet !
+
+Faites-nous part de vos réflexions sur le [serveur Discord](https://discord.gg/CetY6Y4) ou en [créant un ticket](<(https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature&template=feature_request.md&title=)>).
```

Comment:
This looks a little wonky

Reply:
```suggestion
Faites-nous part de vos réflexions sur le [serveur Discord](https://discord.gg/CetY6Y4) ou en [créant un ticket](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature&template=feature_request.md&title=).
```
I think this should fix this

Reply:
Nice catch. This was a problem in all languages 😩 fixed here in #7417